### PR TITLE
Fix(cv_deploy): Fix Ansible Logging and increase API timeouts (#4357)

### PR DESCRIPTION
**Pull request by gmuloc**
## Change Summary

Cherry-pick #4357

### CODE REVIEW

File: ansible_collections/arista/avd/plugins/action/cv_workflow.py [hunk 1]
```diff
@@ -44,44 +44,46 @@
...
+                "options": {
+                    "workspace_build_timeout": {"type": "float", "default": CVTimeOuts.workspace_build_timeout if HAS_PYAVD else 300.0},
+                    "change_control_creation_timeout": {"type": "float", "default": CVTimeOuts.change_control_creation_timeout if HAS_PYAVD else 300.0},
+                },
+                "return_details": {"type": "bool", "required": False, "default": False},
             },
         },
-        "return_details": {"type": "bool", "required": False, "default": False},
-    }
+    )
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/plugins/action/cv_workflow.py]: Why is this changing all over? Looks like an extra parenthesis by mistake.

File: python-avd/pyavd/_cv/client/tag.py [hunk 2]
```diff
@@ -6,6 +6,8 @@
 from datetime import datetime
 from typing import TYPE_CHECKING, Literal
 
+from pyavd._cv.api.arista.time import TimeBounds
+
 from ..api.arista.tag.v2 import (
```
Comment (ClausHolbechArista) [inline python-avd/pyavd/_cv/client/tag.py]:
```suggestion
from pyavd._cv.api.arista.tag.v2 import (
```

Comment (gmuloc) [reply python-avd/pyavd/_cv/client/tag.py]: yeah this one is tough as this wont be consistent with other files in 4.10 as we moved to absolute path later in devel but lets be consistent at file level

File: ansible_collections/arista/avd/plugins/action/cv_workflow.py (hunk 1 above)
Comment (gmuloc) [reply ansible_collections/arista/avd/plugins/action/cv_workflow.py]: let me recheck this - it was causing issue on merge - there is an extra HAS_PYAVD in one of the two files

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**